modules/aws/vpc/main.tf

```diff
@@ -13,23 +13,23 @@ terraform {
   }
 }
 
-resource "aws_flow_log" "flow_log" {
-  iam_role_arn    = aws_iam_role.iam_role[0].arn
-  log_destination = aws_cloudwatch_log_group.log_group[0].arn
+resource "aws_flow_log" "this" {
+  iam_role_arn    = aws_iam_role.this[0].arn
+  log_destination = aws_cloudwatch_log_group.this[0].arn
   traffic_type    = var.vpc_flow_logs_traffic_type
-  vpc_id          = aws_vpc.vpc.id
+  vpc_id          = aws_vpc.this.id
   count           = var.enable_vpc_flow_logs ? 1 : 0
 }
 
 resource "random_uuid" "log_group_guid_identifier" {
 }
 
-resource "aws_cloudwatch_log_group" "log_group" {
+resource "aws_cloudwatch_log_group" "this" {
```

Comment:
should be ``flow_log``
It's an optional feature this and we need to indicate that this only exists if your using flow logs